src/app/common/feature-vtabs.component.ts

```diff
@@ -0,0 +1,148 @@
+import { ComponentReference, GLOBAL_MODULES, CertificatesModuleName, WebServerModuleName, AppPoolsModuleName, WebSitesModuleName, FileSystemModuleName, WebServerModuleIcon } from "main/settings";
+import { Component, Input, OnInit, AfterViewInit, forwardRef, ViewChild } from "@angular/core";
+import { OptionsService } from "main/options.service";
+import { UrlUtil } from "utils/url";
+import { LoggerFactory, Logger, LogLevel } from "diagnostics/logger";
+import { VTabsComponent } from "./vtabs.component";
+import { SectionHelper } from "./section.helper";
+
+export const HomeCategory = "Home";
+export class RouteReference {
+    constructor(public name: string, public ico: string, public routerLink: string[]) {}
+}
+
+export const CONTEXT_MODULES = [
+    new RouteReference(WebServerModuleName, WebServerModuleIcon, [`/webserver/${SectionHelper.normalize(WebServerModuleName)}+general`]),
+    new RouteReference(WebSitesModuleName, "fa fa-globe", [`/webserver/${SectionHelper.normalize(WebSitesModuleName)}`]),
+    new RouteReference(AppPoolsModuleName, "fa fa-cogs", [`/webserver/${SectionHelper.normalize(AppPoolsModuleName)}`]),
+    new RouteReference(FileSystemModuleName, "fa fa-files-o", [`/webserver/${SectionHelper.normalize(FileSystemModuleName)}`]),
+];
+
+export class Feature extends ComponentReference {
+    data: any;
+}
+
+export interface FeatureContext {
+    _links: any
+}
+
+export class GlobalModuleReference {
+    name: string
+    initialize: Promise<boolean>
+}
+
+@Component({
+    selector: 'feature-vtabs',
+    template: `
+<div class="sidebar crumb" [class.nav]="IsActive">
+    <vtabs [markLocation]="true" (activate)="Refresh()" [defaultTab]="default" [categories]="['${HomeCategory}', subcategory]">
+        <item [name]="generalTabName" [ico]="generalTabIcon" [category]="generalTabCategory || subcategory">
+            <ng-content select=".general-tab"></ng-content>
+        </item>
+        <item *ngFor="let module of features" [name]="module.name" [ico]="module.ico" [category]="subcategory">
+            <dynamic [name]="module.component_name" [module]="module" [data]="module.data"></dynamic>
+        </item>
+        <ng-container *ngFor="let module of contexts">
+            <item [name]="module.name" [ico]="module.ico" [category]="'${HomeCategory}'" [routerLink]="module.routerLink">
+                <ng-container *ngIf="!(module.routerLink)">
+                    <dynamic [name]="module.component_name" [module]="module" [data]="module.data"></dynamic>
+                </ng-container>
+            </item>
+        </ng-container>
+    </vtabs>
+</div>
+`,
+    styles: [`
+    :host >>> .sidebar > vtabs .vtabs > .items {
```

Comment:
">>>"  seems to be deprecated according to https://angular.io/guide/component-styles. Please update this not to use the deprecated styles.

Reply:
I can't, removing it seem to move everything out of this section